src/libraries/System.Security.Cryptography/tests/X509Certificates/RevocationTests/AiaTests.cs

```diff
@@ -9,7 +9,7 @@
 namespace System.Security.Cryptography.X509Certificates.Tests.RevocationTests
 {
     [ActiveIssue("https://github.com/dotnet/runtime/issues/57506", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoRuntime), nameof(PlatformDetection.IsMariner))]
-    [SkipOnPlatform(TestPlatforms.Android, "Android does not support AIA fetching")]
+    [SkipOnPlatform(TestPlatforms.Android | TestPlatforms.Browser, "Android nor Browser does not support AIA fetching")]
```

Comment:
Assuming we allow multiple SkipOnPlatform, I'd leave it as two things (browser doesn't support X.509, Android doesn't support AIA)

Reply:
> Assuming we allow multiple SkipOnPlatform

Seems that we have it in many places:

https://github.com/dotnet/runtime/blob/8792a67f1c7340558eea05701c8a92120c5ca87b/src/libraries/Common/tests/StreamConformanceTests/System/IO/StreamConformanceTests.cs#L1634-L1637